perfdash/parser.go

```diff
@@ -286,6 +303,11 @@ func parseSchedulingLatency(data []byte, buildNumber int, testResult *BuildData)
 	testResult.Builds[build] = append(testResult.Builds[build], e2eScheduling)
 	scheduling := parseOperationLatency(obj.SchedulingLatency, "scheduling")
 	testResult.Builds[build] = append(testResult.Builds[build], scheduling)
+
+	for _, ePoint := range extentionsPoints {
+		frameworkExtensionPointDuration := parseOperationLatency(obj.FrameworkExtensionPointDuration[ePoint], fmt.Sprintf("extension_point_duration_%v_latency", strings.ToLower(ePoint)))
```

Comment:
can we please change this to just the extension point name as is. Also, this logic can be done as follows:

```
for name, metric := range obj.FrameworkExtensionPointDuration {
		frameworkExtensionPointDuration := parseOperationLatency(metric, name)
```